quiz.rb

```diff
@@ -2,19 +2,60 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+
+  attr_reader :name
+  attr_reader :breed
+  attr_reader :age
+
+  def initialize(name,breed,age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+
+end
+
 # Question 2
 # Create a new Dog object.
 
+parker = Dog.new("Parker","Eskimo Lab","4")
+
 # Question 3
 # Create a Person class, with attributes name, age, and location.
 # It should set all three attributes when you create a new Person.
 # Name and location should be writeable, but age should be read-only.
 
+class Person
+
+  attr_writer :name
```

Comment:
Not wrong, but you'd generally want to use `attr_accessor` rather than `attr_writer`, since `attr_writer` will only give you write access, _not_ read access, and it's rare that you'd want someone to be able to change something but not be able to see it.